docs/timeline/index.html

```diff
@@ -469,6 +469,12 @@ <h3 id="groups">Groups</h3>
       <td>no</td>
       <td>Assuming the group has nested groups, this will set the initial state of the group - shown or collapsed. The <code>showNested</code> is defaulted to <code>true</code>.</td>
     </tr>
+    <tr>
+      <td>heightMode</td>
+      <td>String</td>
+      <td>no</td>
+      <td>This field is optional. If set this overrides the global <code>groupHeightMode</code> configuration option for this group.</td>
```

Comment:
Can you please all the valid option ('auto' and 'fixed') to this description.

Reply:
Sure, done.